packages/std/src/lib.rs

```diff
@@ -45,8 +45,9 @@ pub use crate::coins::Coins;
 pub use crate::deps::{Deps, DepsMut, OwnedDeps};
 pub use crate::errors::{
     CheckedFromRatioError, CheckedMultiplyFractionError, CheckedMultiplyRatioError,
-    ConversionOverflowError, DivideByZeroError, OverflowError, OverflowOperation,
-    RecoverPubkeyError, StdError, StdResult, SystemError, VerificationError,
+    CoinFromStrError, CoinsError, ConversionOverflowError, DivideByZeroError, DivisionError,
+    OverflowError, OverflowOperation, RecoverPubkeyError, StdError, StdResult, SystemError,
+    VerificationError,
```

Comment:
Shouldn't this diff disappear when we merge the latest patch release into main first?

Reply:
Yes, I will merge 1.3 into main and create a PR for that. We can also rebase this PR to remove the commit with the exports